packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -411,7 +415,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   const { openVideoEffectsPane } = useVideoEffectsPane(
     props.updateSidePaneRenderer,
     props.mobileView,
-    props.latestErrors,
+    props.latestErrors as ActiveErrorMessage[],
```

Comment:
Do we need this casting?

Reply:
Yeah videoeffects only take ActiveErrorMessage[] not ActiveNotification[]